ledger-keyring-mv2.js

```diff
@@ -0,0 +1,195 @@
+const { BaseLedgerKeyring } = require('./base-ledger-keyring')
+
+const CONNECTION_EVENT = 'ledger-connection-change'
+
+class LedgerKeyringMv2 extends BaseLedgerKeyring {
```

Comment:
Kind of trivial, but is there a better name that doesn't refer to a manifest version? Maybe `LedgerKeyringIframe`? I say this because:
* Iframes can still be set up on MV3 via client-side pages
* AFAIK, this package is designed to be env agnostic, e.g. someone might be using this package within a website? Or some other environment where manifest versions aren't applicable